### PR TITLE
Fixes #8307: Use variables for path to cf-agent in system Techniques to work on Windows

### PR DESCRIPTION
**Pull request by jooooooon**
https://www.rudder-project.org/redmine/issues/8307

### CODE REVIEW

**Comment by jooooooon** [comment]
@ncharles Could you review this please?

File: initial-promises/node-server/common/1.0/cf-served.cf [hunk 1]
```diff
@@ -60,7 +60,7 @@ bundle server access_rules
 
     any::
       # Allow server to remotely run the agent
-      "/var/rudder/cfengine-community/bin/cf-agent"
+      "${sys.cf-agent}"
```
Comment (ncharles) [inline initial-promises/node-server/common/1.0/cf-served.cf]: the variable is ${sys.cf_agent} (note the underscore)

**Comment by jooooooon** [comment]
PR rebased

File: initial-promises/node-server/common/1.0/cf-served.cf (hunk 1 above)
Comment (jooooooon) [inline initial-promises/node-server/common/1.0/cf-served.cf]: Good catch! Fixed.

**Comment by jooooooon** [comment]
Commit modified

**Comment by ncharles** [comment]
merging

**Pull request merged**